### ISSUE TITLE
Update to 15.10

### ISSUE THREAD

**Issue by smlavoie** [geoffrowland/mahara-artefact_cpds#12]
We're using CPDS plugin since 2011 and it is one big incentive for my teachers to use it. 

We upgraded to 15.10 but some images links are broken (edit, delete, etc).
![cpd](https://cloud.githubusercontent.com/assets/2454560/12449010/30156264-bf48-11e5-8852-c72d35175a5a.PNG)
Will you update the plugin to fit in 15.10?

**Comment by geoffrowland** [comment]
Hi

Good to hear your institution and teachers find the CPDs plugin useful. 

I'll lake a look at the broken link-images and roll-out a fix ASAP. Based on experience of similar issues with previous Mahara updates, will hopefully only take a few days.

Thanks for your feedback.

**Comment by geoffrowland** [comment]
Yes, have confirmed the issue with Mahara 15.10. Had only tested with 15.04 until recently.
Although the CPDs artefact retains basic functionality, link-images/buttons are broken. Am working on a fix restoring link-images/buttons similar to the Plans artefact.

**Comment by smlavoie** [comment]
Super! Thanks a lot!

**Comment by ngson2000** [comment]
Hi Geoff,

We've update the plugin to work with mahara 15.10. Please review our upstream code https://github.com/geoffrowland/mahara-artefact_cpds/pull/13

**Comment by geoffrowland** [comment]
Hi Son

I have noticed your CPDs plugin updates for Mahara 15.10 have introduced a couple of 'reversions' and just wanted to check if the implications of these. I appreciate that these are now consistent with Plans.

1. You have made the Description field for CPDs and Activities to be a basic textarea control, rather than a rich text WYSIWYG (TinyMCE) control. I changed this, a while back, in response to user requests for the ability to enter more 'complex' Descriptions e.g. hyperlinks to related resources. I am ambivalent about this and can also see the 'KISS' appeal of a basic textarea Description. Links/resources can always be added elsewhere in, say, a Mahara Page containing the CPD log. However, I don't want to upset users by breaking display of existing rich content in the Descriptions.

2. You have reduced the pagination limit of Activities from 20 back to 10. This introduces an issue in that if the number of Activities exceeds 10, which may be quite likely for, say, the CPD log for an academic year, the calculated total hours may be misleading as they only give the total for that particular page. Ideally, should perhaps have a page subtotal and a grand total - but I have not (yet) worked out how to implement this. So, as a rough-and-ready workaround, had just increased the limit to 20 to reduce (but not eliminate) the chance of multiple pages and subtotals.

So, before I revert your reversions, any comments?

Again, thanks for the update. It has saved me considerable work!
